File: src/programmersguide/asposecells/workingwithfiles/utilityfeatures/convertingtomhtmlfiles/java/ConvertingToMHTMLFiles.java
/* 
 * Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
 *
 * This file is part of Aspose.Cells. The source code in this file
 * is only intended as a supplement to the documentation, and is provided
 * "as is", without warranty of any kind, either expressed or implied.
 */
 
package programmersguide.asposecells.workingwithfiles.utilityfeatures.convertingtomhtmlfiles.java;

import com.aspose.cells.HtmlSaveOptions;
import com.aspose.cells.SaveFormat;
import com.aspose.cells.Workbook;

public class ConvertingToMHTMLFiles
{
    public static void main(String[] args) throws Exception
    {
        // The path to the documents directory.
        String dataDir = "src/programmersguide/asposecells/workingwithfiles/utilityfeatures/convertingtomhtmlfiles/data/";

        //Specify the file path
        String filePath = dataDir + "Book1.xlsx";

        //Specify the HTML saving options
        HtmlSaveOptions sv = new HtmlSaveOptions(SaveFormat.M_HTML);

        //Instantiate a workbook and open the template XLSX file
        Workbook wb = new Workbook(filePath);

        //Save the MHT file
        wb.save(filePath + ".out.mht", sv);

        // Print message
        System.out.println("Excel to MHTML conversion performed successfully.");
    }
}

File: Plugins/Aspose_Cells_Java_for_Jython/asposecells/WorkingWithFiles/ConvertingToMhtmlFiles.py
from asposecells import Settings
from com.aspose.cells import Workbook
from com.aspose.cells import HtmlSaveOptions
from com.aspose.cells import SaveFormat


class ConvertingToMhtmlFiles:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithFiles/ConvertingToMhtmlFiles/'
        
        saveFormat = SaveFormat

        #Specify the file path
        filePath = dataDir + "Book1.xlsx"

        #Specify the HTML saving options
        sv = HtmlSaveOptions(saveFormat.M_HTML)

        #Instantiate a workbook and open the template XLSX file
        wb = Workbook(filePath)

        #Save the MHT file
        wb.save(filePath + ".out.mht", sv)

        # Print message
        print "Excel to MHTML conversion performed successfully."
 
if __name__ == '__main__':        
    ConvertingToMhtmlFiles()